pkg/controllers/clusterset/globalclusterset/global_clusterset_controller.go

```diff
@@ -0,0 +1,167 @@
+package globalclusterset
+
+import (
+	"context"
```

Comment:
why we cannot do it in clusterset_controller?

Reply:
moved